ice_in

```diff
@@ -1,175 +1,457 @@
-  f_sig1 = 'x', f_sig2 = 'x'
-  f_sigP = 'x'
-  f_snow = 'x'
-  f_sst = 'x'
-  f_sss = 'x'
-  f_fswabs = 'x'
-  f_taubx = 'x' , f_tauby = 'x'
-  f_Tref = 'x', f_Qref = 'x'
-  f_uocn = 'x' , f_vocn = 'x'
-/
\ No newline at end of file
+  f_aice = "mdhxx"
+  f_aicen = "mdhxx"
+  f_aisnap = "xxxxx"
+  f_albice = "xxxxx"
+  f_albpnd = "xxxxx"
+  f_albsni = "mxxxx"
+  f_albsno = "xxxxx"
+  f_alidf = "xxxxx"
+  f_alidf_ai = "mxxxx"
+  f_alidr = "xxxxx"
+  f_alidr_ai = "mxxxx"
+  f_alvdf = "xxxxx"
+  f_alvdf_ai = "mxxxx"
+  f_alvdr = "xxxxx"
+  f_alvdr_ai = "mxxxx"
+  f_angle = .true.
+  f_anglet = .true.
+  f_blkmask = .false.
+  f_bounds = .true.
+  f_cmip = "xxxxx"
+  f_congel = "mdxxx"
+  f_coszen = "xxxxx"
+  f_daidtd = "mdxxx"
+  f_daidtt = "mdxxx"
+  f_divu = "mxxxx"
+  f_dsnow = "xxxxx"
+  f_dvidtd = "mdxxx"
+  f_dvidtt = "mdxxx"
+  f_dxt = .true.
+  f_dxu = .true.
+  f_dyt = .true.
+  f_dyu = .true.
+  f_evap = "mxxxx"
+  f_evap_ai = "xxxxx"
+  f_fcondtop_ai = "xxxxx"
+  f_fcondtopn_ai = "xxxxx"
+  f_fhocn = "mxxxx"
+  f_fhocn_ai = "xxxxx"
+  f_flat = "mxxxx"
+  f_flat_ai = "mxxxx"
+  f_flatn_ai = "xxxxx"
+  f_flwdn = "mxxxx"
+  f_flwup = "mxxxx"
+  f_flwup_ai = "xxxxx"
+  f_fmeltt_ai = "xxxxx"
+  f_fmelttn_ai = "xxxxx"
+  f_frazil = "mdxxx"
+  f_fresh = "mxxxx"
+  f_fresh_ai = "xxxxx"
+  f_frz_onset = "xxxxx"
+  f_frzmlt = "xxxxx"
+  f_fsalt = "mxxxx"
+  f_fsalt_ai = "xxxxx"
+  f_fsens = "mxxxx"
+  f_fsens_ai = "mxxxx"
+  f_fsensn_ai = "xxxxx"
+  f_fsurf_ai = "xxxxx"
+  f_fsurfn_ai = "xxxxx"
+  f_fswabs = "mdhxx"
+  f_fswabs_ai = "xxxxx"
+  f_fswdn = "mdhxx"
+  f_fswfac = "xxxxx"
+  f_fswint_ai = "xxxxx"
+  f_fswthru = "mdhxx"
+  f_fswthru_ai = "xxxxx"
+  f_fswup = "mxxxx"
+  f_fy = "xxxxx"
+  f_hi = "mdhxx"
+  f_hisnap = "xxxxx"
+  f_hs = "mdhxx"
+  f_hte = .true.
+  f_htn = .true.
+  f_iage = "xxxxx"
+  f_icepresent = "mxxxx"
+  f_keffn_top = "xxxxx"
+  f_meltb = "mdxxx"
+  f_meltl = "mdxxx"
+  f_melts = "mdxxx"
+  f_meltt = "mdxxx"
+  f_mlt_onset = "xxxxx"
+  f_ncat = .true.
+  f_qref = "xxxxx"
+  f_rain = "mxxxx"
+  f_rain_ai = "xxxxx"
+  f_shear = "mxxxx"
+  f_sice = "xxxxx"
+  f_sig1 = "mxxxx"
+  f_sig2 = "mxxxx"
+  f_sinz = "xxxxx"
+  f_snoice = "mxxxx"
+  f_snow = "mxxxx"
+  f_snow_ai = "xxxxx"
+  f_snowfrac = "mxxxx"
+  f_snowfracn = "xxxxx"
+  f_sss = "xxxxx"
+  f_sst = "xxxxx"
+  f_strairx = "mxxxx"
+  f_strairy = "mxxxx"
+  f_strcorx = "mxxxx"
+  f_strcory = "mxxxx"
+  f_strength = "mxxxx"
+  f_strintx = "mxxxx"
+  f_strinty = "mxxxx"
+  f_strocnx = "mxxxx"
+  f_strocny = "mxxxx"
+  f_strtltx = "mxxxx"
+  f_strtlty = "mxxxx"
+  f_tair = "xxxxx"
+  f_tarea = .true.
+  f_tinz = "xxxxx"
+  f_tmask = .true.
+  f_tref = "xxxxx"
+  f_trsig = "xxxxx"
+  f_tsfc = "mxxxx"
+  f_tsnz = "xxxxx"
+  f_uarea = .true.
+  f_uatm = "mxxxx"
+  f_uocn = "xxxxx"
+  f_uvel = "mxxxx"
+  f_vatm = "mxxxx"
+  f_vgrdb = .true.
+  f_vgrdi = .true.
+  f_vgrds = .true.
+  f_vicen = "mdhxx"
+  f_vocn = "xxxxx"
+  f_vsnon = "mdhxx"
+  f_vvel = "mxxxx"
+/
```

Comment:
I don't understand why there are so many changes here? My [diff](https://github.com/COSIMA/access-om3/issues/166#issue-2301785234) showed there were only a few differences related to `fsd`

Reply:
Resolved in this commit d260731